packages/www/app/routes/_index.tsx

```diff
@@ -20,12 +20,46 @@ export const meta: MetaFunction = () => {
 
 export default function Index() {
   const [stats, setStats] = useState({
-    stars: 123,
-    contributors: 123,
-    nodes: 123,
-    workflows: 123,
+    stars: 0,
+    contributors: 0,
+    blocks: 0,
+    workflows: 0,
   });
 
+  useEffect(() => {
+    const fetchRepoData = async () => {
+      try {
+        const repoUrl = "https://api.github.com/repos/softflow24/data-river";
+
+        const repoResponse = await fetch(repoUrl);
+        if (!repoResponse.ok) {
+          throw new Error(
+            "Error fetching repository data: ${repoResponse.statusText}",
+          );
+        }
+
+        const contributorsResponse = await fetch(repoUrl + "/contributors");
+        if (!contributorsResponse.ok) {
+          throw new Error(
+            "Error fetching repository data: ${contributorsResponse.statusText}",
+          );
+        }
+        const repoData = await repoResponse.json();
+        const contributorsData = await contributorsResponse.json();
+        setStats({
+          stars: repoData.stargazers_count,
+          contributors: contributorsData.length,
+          blocks: 5,
+          workflows: 0,
+        });
+      } catch (error: any) {
+        console.log(error.message);
```

Comment:
```suggestion
        console.error(error.message);
```
In order to log at the correct error level.